src/dmd/hdrgen.d

```diff
@@ -3411,6 +3411,8 @@ private void visitFuncIdentWithPostfix(TypeFunction t, const char[] ident, OutBu
         linkageToBuffer(buf, t.linkage);
         buf.writeByte(' ');
     }
+    if (t.linkage == LINK.objc && isStatic)
+        buf.write("static ");
```

Comment:
why do other static member functions not need this?

Reply:
I have not been able to come up with a case were this would occur for non-objc methods.